src/app/projects/project-progress-dashboard/project-progress-dashboard.tpl 2.html

```diff
@@ -0,0 +1,87 @@
+<div ng-if="unit" class="project-progress-dashboard">
+  <div class="col-md-4 col-md-push-8">
+    <div class="row">
+      <div class="col-sm-6 col-md-12">
+        <div class="panel panel-primary">
+          <div class="panel-heading clearfix">
+            <div class="pull-left">
+              <h4 class="panel-title">Task List</h4>
+            </div>
+          </div>
+          <div class="panel-body">
+            <project-tasks-list project="project" student-project-id="studentProjectId" task-def="taskDef" unit="unit" assessing-unit-role="assessingUnitRole" on-select="showTaskView"></project-tasks-list>
+          </div>
+        </div><!--task-lists-->
+      </div>
+      <div class="col-sm-6 col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h3 class="panel-title">Target Grade</h3>
+            Select the grade you wish to achieve in the unit.
+          </div>
+          <div class="panel-body text-center">
+            <p class="btn-group">
+              <label ng-repeat="grade in grades" ng-if="$index != 4" ng-click="chooseGrade($index)" class="btn btn-default col-sm-3 text-center" ng-model="project.targetGrade" btn-radio="{{$index}}">
+                <grade-icon grade="grade" tooltip="Select a {{grade}} as your target grade" tooltip-append-to-body="true" class="text-{{$index == project.targetGrade ? 'primary' : 'muted'}}"></grade-icon>
```

Comment:
use Material Design Tooltips:
https://v6.material.angular.io/components/tooltip/examples

Reply:
Not sure if its in my scope. I was migrating project-tasks-list to angular so only focused on the component. I think i might have commited extra file for some reason.